In [6]:
import pandas as pd
import numpy as np
import warnings
warnings.filterwarnings("ignore")


In [ ]:
londonward_socio = 

In [18]:
df_district_code = pd.read_csv("London District codes(1).csv")
df_district_code

District Districtcode
0     Barking and Dagenham         00AB
1                   Barnet         00AC
2                   Bexley         00AD
3                    Brent         00AE
4                  Bromley         00AF
5                   Camden         00AG
6                  Croydon         00AH
7                   Ealing         00AJ
8                  Enfield         00AK
9                Greenwich         00AL
10                 Hackney         00AM
11  Hammersmith and Fulham         00AN
12                Haringey         00AP
13                  Harrow         00AQ
14                Havering         00AR
15              Hillingdon         00AS
16                Hounslow         00AT
17               Islington         00AU
18  Kensington and Chelsea         00AW
19    Kingston upon Thames         00AX
20                 Lambeth         00AY
21                Lewisham         00AZ
22                  Merton         00BA
23                  Newham         00BB
24               Redbridge         00BC
25    Richmond upon Thames         00BD
26               Southwark         00BE
27                  Sutton         00BF
28           Tower Hamlets         00BG
29          Waltham Forest         00BH
30              Wandsworth         00BJ
31             Westminster         00BK

In [7]:
info=pd.DataFrame(df_dictrict_code.dtypes).T.rename(index={0:'Data type'})
info=info.append(pd.DataFrame(df_dictrict_code.isna().sum()).T.rename(index={0:'Null values count'}))
info=info.append(pd.DataFrame(df_dictrict_code.isna().sum()/df_dictrict_code.shape[0]*100).T.
                         rename(index={0:'Null values (%)'}))
info=info.append(pd.DataFrame(df_dictrict_code.min()).T.rename(index={0:'Min values'}))
info=info.append(pd.DataFrame(df_dictrict_code.max()).T.rename(index={0:'Max values'}))
info

District Districtcode
Data type                        object       object
Null values count                     0            0
Null values (%)                     0.0          0.0
Min values         Barking and Dagenham         00AB
Max values                  Westminster         00BK

In [16]:
df_demographics = pd.read_csv("London ward data demographics(1).csv", sep='\t')
df_demographics

Wardname   Children  Greaterthan65  nonwhite  \
0                 Hackney - Queensbridge  17.542063            8.2      44.7   
1         Hammersmith & Fulham - S&s End  17.915361            8.4      30.6   
2             Barking & Dagenham - River  26.851598           10.1      38.4   
3    Tower Hamlets - Bethnal Green North  18.555872            7.0      49.4   
4                         Merton - Abbey  15.731861            8.5      26.6   
..                                   ...        ...            ...       ...   
618                 Haringey - Harringay  14.767932            7.5      34.5   
619                Lambeth - Coldharbour  21.562202            6.1      62.1   
620              Newham - Plaistow North  23.600517            7.1      73.4   
621               Greenwich - Abbey Wood  26.853031            9.6      46.3   
622         Southwark - South Camberwell  19.684852            7.1      44.1   

     NotBorninUK  NotEnglishspeaking  
0           38.3                11.8  
1           39.0                11.8  
2           30.9                 9.6  
3           38.7                15.0  
4           38.3                12.5  
..           ...                 ...  
618         49.2                24.3  
619         43.7                13.3  
620         52.1                23.6  
621         32.0                 7.6  
622         35.8                 7.7  

[623 rows x 6 columns]

| Variable Name | Description |
|:-------------------------------:|:-----------:|
|       Wardname                 |  Wardname         |
| Children | |
| Greeaterthan65 | % All Older people aged 65+ - 2011 |
| nonwhite | % BAME - 2011 |
| NotBorninUK | % Not Born in UK - 2011 |
| NotEnglishspeaking | % English is First Language of no one in the household - 2011 | 

In [56]:
info=pd.DataFrame(df_demographics.dtypes).T.rename(index={0:'Data type'})
info=info.append(pd.DataFrame(df_demographics.isna().sum()).T.rename(index={0:'Null values count'}))
info=info.append(pd.DataFrame(df_demographics.isna().sum()/df_demographics.shape[0]*100).T.
                         rename(index={0:'Null values (%)'}))
info=info.append(pd.DataFrame(df_demographics.min()).T.rename(index={0:'Min values'}))
info=info.append(pd.DataFrame(df_demographics.max()).T.rename(index={0:'Max values'}))
info

Wardname   Children Greaterthan65  \
Data type                              object    float64       float64   
Null values count                           0          1             1   
Null values (%)                           0.0   0.160514      0.160514   
Min values         Barking & Dagenham - Abbey   7.133676           3.5   
Max values           Westminster - Westbourne  33.187914          23.5   

                   nonwhite NotBorninUK NotEnglishspeaking  
Data type           float64     float64            float64  
Null values count         1           1                  1  
Null values (%)    0.160514    0.160514           0.160514  
Min values              4.1         5.1                0.4  
Max values             93.7        68.5               99.7

In [36]:
df_demographics[df_demographics['Children'].isnull()]

Wardname  Children  \
521  Richmond upon Thames - Ham, Petersham & Richmo...       NaN   

     Greaterthan65  nonwhite  NotBorninUK  NotEnglishspeaking  
521            NaN       NaN          NaN                 NaN

In [37]:
df_environment = pd.read_csv("London ward data environment(1).csv")
df_environment

Wardcode  Population2011Census  Crimerate  Openspace
0     00ANGQ                 11201      117.7        0.0
1     00ANGA                 11518      114.0        0.3
2     00ADGN                 10800       44.2        0.7
3     00BEGH                 12321       65.3        0.7
4     00BCFZ                 12609       64.3        1.3
..       ...                   ...        ...        ...
619   00BDFZ                 10317       65.8       81.9
620   00ARGW                 12833       49.5       82.0
621   00ASGN                  7399       67.4       85.5
622   00AFGQ                  5110       58.6       88.8
623   00AZGW                 16414       71.7       99.9

[624 rows x 4 columns]

| Variable Name | Description |
|:-------------------------------:|:-----------:|
|       Crimerate                 | Crime rate - 2011/12         |
| Population2011Cencus | Population 2011 (Cencus) |
| Openspace | % area that is open space - 2012 |

In [57]:
info=pd.DataFrame(df_environment.dtypes).T.rename(index={0:'Data type'})
info=info.append(pd.DataFrame(df_environment.isna().sum()).T.rename(index={0:'Null values count'}))
info=info.append(pd.DataFrame(df_environment.isna().sum()/df_environment.shape[0]*100).T.
                         rename(index={0:'Null values (%)'}))
info=info.append(pd.DataFrame(df_environment.min()).T.rename(index={0:'Min values'}))
info=info.append(pd.DataFrame(df_environment.max()).T.rename(index={0:'Max values'}))
info

Wardcode Population2011Census Crimerate Openspace
Data type           object                int64   float64   float64
Null values count        0                    0         0         0
Null values (%)        0.0                  0.0       0.0       0.0
Min values          00ABFX                 5110       0.9       0.0
Max values          00BKGW                23084    2100.2      99.9

In [66]:
df_health = pd.read_csv("london ward data health(1).csv")
df_health

Unnamed: 0                                   Wardname  \
0             0                           Bromley - Darwin   
1             1    Kensington and Chelsea - Royal Hospital   
2             2                     Hillingdon - Harefield   
3             3  Hammersmith and Fulham - Palace Riverside   
4             4         Kensington and Chelsea - Pembridge   
..          ...                                        ...   
617         617             Greenwich - Woolwich Riverside   
618         618  Tower Hamlets - Blackwall and Cubitt Town   
619         619            Greenwich - Thamesmead Moorings   
620         620                       Barnet - Childs Hill   
621         621                   Tower Hamlets - Millwall   

     Population2011Census  GeneralFertilityRate  Malelifeexpectancy  \
0                  5110.0                  63.8                81.2   
1                  7252.0                  52.3                80.5   
2                  7399.0                  55.8                78.3   
3                  7483.0                  40.0                80.9   
4                  7659.0                  40.4                82.1   
..                    ...                   ...                 ...   
617               19120.0                  84.4                73.8   
618               19461.0                  75.1                77.4   
619               19730.0                 122.5                74.3   
620               20049.0                  69.0                77.4   
621               23084.0                  58.1                83.6   

     Femalelifeexpectancy  
0                    82.4  
1                    89.6  
2                    82.3  
3                    89.0  
4                    85.7  
..                    ...  
617                  80.2  
618                  83.4  
619                  77.5  
620                  82.0  
621                  95.0  

[622 rows x 6 columns]

| Variable Name | Description |
|:-------------------------------:|:-----------:|
| GeneralFertilityRate | General Fertility Rate - 2011 |
| Malelifeexpectancy | Male life expectancy - 2005-2009 |
| Femalelifeexpectancy | Female life expectancy - 2005-2009 |

In [67]:
df_health = df_health.drop('Unnamed: 0', axis=1)
df_health

Wardname  Population2011Census  \
0                             Bromley - Darwin                5110.0   
1      Kensington and Chelsea - Royal Hospital                7252.0   
2                       Hillingdon - Harefield                7399.0   
3    Hammersmith and Fulham - Palace Riverside                7483.0   
4           Kensington and Chelsea - Pembridge                7659.0   
..                                         ...                   ...   
617             Greenwich - Woolwich Riverside               19120.0   
618  Tower Hamlets - Blackwall and Cubitt Town               19461.0   
619            Greenwich - Thamesmead Moorings               19730.0   
620                       Barnet - Childs Hill               20049.0   
621                   Tower Hamlets - Millwall               23084.0   

     GeneralFertilityRate  Malelifeexpectancy  Femalelifeexpectancy  
0                    63.8                81.2                  82.4  
1                    52.3                80.5                  89.6  
2                    55.8                78.3                  82.3  
3                    40.0                80.9                  89.0  
4                    40.4                82.1                  85.7  
..                    ...                 ...                   ...  
617                  84.4                73.8                  80.2  
618                  75.1                77.4                  83.4  
619                 122.5                74.3                  77.5  
620                  69.0                77.4                  82.0  
621                  58.1                83.6                  95.0  

[622 rows x 5 columns]

In [68]:
info=pd.DataFrame(df_health.dtypes).T.rename(index={0:'Data type'})
info=info.append(pd.DataFrame(df_health.isna().sum()).T.rename(index={0:'Null values count'}))
info=info.append(pd.DataFrame(df_health.isna().sum()/df_health.shape[0]*100).T.
                         rename(index={0:'Null values (%)'}))
info=info.append(pd.DataFrame(df_health.min()).T.rename(index={0:'Min values'}))
info=info.append(pd.DataFrame(df_health.max()).T.rename(index={0:'Max values'}))
info

Wardname Population2011Census  \
Data type                                object              float64   
Null values count                             0                    0   
Null values (%)                             0.0                  0.0   
Min values         Barking and Dagenham - Abbey               5110.0   
Max values             Westminster - Westbourne              23084.0   

                  GeneralFertilityRate Malelifeexpectancy Femalelifeexpectancy  
Data type                      float64            float64              float64  
Null values count                    0                  0                    0  
Null values (%)                    0.0                0.0                  0.0  
Min values                        22.1               71.2                 75.9  
Max values                       130.2              178.0                 99.6

In [69]:
temp = df_health["Wardname"].str.split(" - ", n = 1, expand = True)
 
df_health["District"]= temp[0]
 
# making separate last name column from new data frame
df_health["Ward name"]= temp[1]
 
# Dropping old Name columns
df_health.drop(columns =["Wardname"], inplace = True)
 
df_health

Population2011Census  GeneralFertilityRate  Malelifeexpectancy  \
0                  5110.0                  63.8                81.2   
1                  7252.0                  52.3                80.5   
2                  7399.0                  55.8                78.3   
3                  7483.0                  40.0                80.9   
4                  7659.0                  40.4                82.1   
..                    ...                   ...                 ...   
617               19120.0                  84.4                73.8   
618               19461.0                  75.1                77.4   
619               19730.0                 122.5                74.3   
620               20049.0                  69.0                77.4   
621               23084.0                  58.1                83.6   

     Femalelifeexpectancy                District                  Ward name  
0                    82.4                 Bromley                     Darwin  
1                    89.6  Kensington and Chelsea             Royal Hospital  
2                    82.3              Hillingdon                  Harefield  
3                    89.0  Hammersmith and Fulham           Palace Riverside  
4                    85.7  Kensington and Chelsea                  Pembridge  
..                    ...                     ...                        ...  
617                  80.2               Greenwich         Woolwich Riverside  
618                  83.4           Tower Hamlets  Blackwall and Cubitt Town  
619                  77.5               Greenwich        Thamesmead Moorings  
620                  82.0                  Barnet                Childs Hill  
621                  95.0           Tower Hamlets                   Millwall  

[622 rows x 6 columns]

Population2011Census in df_health and df_environment are not in the same data type, so the one in df_health will be turnen into int64.

In [71]:
df_health['Population2011Census'] = df_health['Population2011Census'].astype(int)
info=pd.DataFrame(df_health.dtypes).T.rename(index={0:'Data type'})
info=info.append(pd.DataFrame(df_health.isna().sum()).T.rename(index={0:'Null values count'}))
info=info.append(pd.DataFrame(df_health.isna().sum()/df_health.shape[0]*100).T.
                         rename(index={0:'Null values (%)'}))
info=info.append(pd.DataFrame(df_health.min()).T.rename(index={0:'Min values'}))
info=info.append(pd.DataFrame(df_health.max()).T.rename(index={0:'Max values'}))
info

Population2011Census GeneralFertilityRate  \
Data type                        int32              float64   
Null values count                    0                    0   
Null values (%)                    0.0                  0.0   
Min values                        5110                 22.1   
Max values                       23084                130.2   

                  Malelifeexpectancy Femalelifeexpectancy  \
Data type                    float64              float64   
Null values count                  0                    0   
Null values (%)                  0.0                  0.0   
Min values                      71.2                 75.9   
Max values                     178.0                 99.6   

                               District Ward name  
Data type                        object    object  
Null values count                     0         0  
Null values (%)                     0.0       0.0  
Min values         Barking and Dagenham     Abbey  
Max values                  Westminster  Yiewsley

In [72]:
df_health[df_health['Population2011Census'] == 11201]

Population2011Census  GeneralFertilityRate  Malelifeexpectancy  \
156                 11201                  47.3                78.8   

     Femalelifeexpectancy                District Ward name  
156                  81.1  Hammersmith and Fulham      Town

In [73]:
df_environment[df_environment['Population2011Census'] == 11201]

Wardcode  Population2011Census  Crimerate  Openspace
0   00ANGQ                 11201      117.7        0.0

There are 622 data in `df_health` and 624 data in `df_environment`. So if both tables are joined on `Popolation2011Census`, there will be 2 null values the new joined table.

In [100]:
envi_health=pd.merge(df_health,df_environment,on='Population2011Census', how='right')
info=pd.DataFrame(envi_health.dtypes).T.rename(index={0:'Data type'})
info=info.append(pd.DataFrame(envi_health.isna().sum()).T.rename(index={0:'Null values count'}))
info

Population2011Census GeneralFertilityRate  \
Data type                        int64              float64   
Null values count                    0                    2   

                  Malelifeexpectancy Femalelifeexpectancy District Ward name  \
Data type                    float64              float64   object    object   
Null values count                  2                    2        2         2   

                  Wardcode Crimerate Openspace  
Data type           object   float64   float64  
Null values count        0         0         0

In [101]:
envi_health[envi_health['GeneralFertilityRate'].isna()]

Population2011Census  GeneralFertilityRate  Malelifeexpectancy  \
149                 14777                   NaN                 NaN   
257                 16544                   NaN                 NaN   

     Femalelifeexpectancy District Ward name Wardcode  Crimerate  Openspace  
149                   NaN      NaN       NaN   00AYGN       94.4       13.2  
257                   NaN      NaN       NaN   00BCGL      108.0       18.3

There are 2 empty rows in the joined table, as shown above. \n
`00AYGN` = Lambeth - Stockwell
`00BCGL` = Redbridge - Loxford

Source: https://data.london.gov.uk/dataset/ward-profiles-and-atlas

In [102]:
envi_health

Population2011Census  GeneralFertilityRate  Malelifeexpectancy  \
0                   11201                  47.3                78.8   
1                   11518                  49.4                78.8   
2                   10800                  58.1                78.2   
3                   12321                  76.7                81.2   
4                   12609                  68.7                81.0   
..                    ...                   ...                 ...   
667                 10317                  68.8                78.3   
668                 12833                  38.6                80.1   
669                  7399                  55.8                78.3   
670                  5110                  63.8                81.2   
671                 16414                  53.8                76.4   

     Femalelifeexpectancy                District  \
0                    81.1  Hammersmith and Fulham   
1                    88.6  Hammersmith and Fulham   
2                    82.8                  Bexley   
3                    82.9               Southwark   
4                    85.7               Redbridge   
..                    ...                     ...   
667                  86.4    Richmond upon Thames   
668                  81.6                Havering   
669                  82.3              Hillingdon   
670                  82.4                 Bromley   
671                  78.8                Lewisham   

                               Ward name Wardcode  Crimerate  Openspace  
0                                   Town   00ANGQ      117.7        0.0  
1                                Addison   00ANGA      114.0        0.3  
2                 Falconwood and Welling   00ADGN       44.2        0.7  
3                           East Dulwich   00BEGH       65.3        0.7  
4                            Barkingside   00BCFZ       64.3        1.3  
..                                   ...      ...        ...        ...  
667  Ham, Petersham & Richmond Riverside   00BDFZ       65.8       81.9  
668                            Upminster   00ARGW       49.5       82.0  
669                            Harefield   00ASGN       67.4       85.5  
670                               Darwin   00AFGQ       58.6       88.8  
671                       Telegraph Hill   00AZGW       71.7       99.9  

[672 rows x 9 columns]

In [88]:
temp = df_demographics["Wardname"].str.split(" - ", n = 1, expand = True)
 
df_demographics["District"]= temp[0]
 
# making separate last name column from new data frame
df_demographics["Ward name"]= temp[1]
 
# Dropping old Name columns
df_demographics.drop(columns =["Wardname"], inplace = True)
 
df_demographics

Children  Greaterthan65  nonwhite  NotBorninUK  NotEnglishspeaking  \
0    17.542063            8.2      44.7         38.3                11.8   
1    17.915361            8.4      30.6         39.0                11.8   
2    26.851598           10.1      38.4         30.9                 9.6   
3    18.555872            7.0      49.4         38.7                15.0   
4    15.731861            8.5      26.6         38.3                12.5   
..         ...            ...       ...          ...                 ...   
618  14.767932            7.5      34.5         49.2                24.3   
619  21.562202            6.1      62.1         43.7                13.3   
620  23.600517            7.1      73.4         52.1                23.6   
621  26.853031            9.6      46.3         32.0                 7.6   
622  19.684852            7.1      44.1         35.8                 7.7   

                 District            Ward name  
0                 Hackney         Queensbridge  
1    Hammersmith & Fulham              S&s End  
2      Barking & Dagenham                River  
3           Tower Hamlets  Bethnal Green North  
4                  Merton                Abbey  
..                    ...                  ...  
618              Haringey            Harringay  
619               Lambeth          Coldharbour  
620                Newham       Plaistow North  
621             Greenwich           Abbey Wood  
622             Southwark     South Camberwell  

[623 rows x 7 columns]

In [90]:
envi_health_demo=pd.merge(envi_health,df_demographics,on='Ward name', how='left')
info=pd.DataFrame(envi_health_demo.dtypes).T.rename(index={0:'Data type'})
info=info.append(pd.DataFrame(envi_health_demo.isna().sum()).T.rename(index={0:'Null values count'}))
info

Population2011Census GeneralFertilityRate  \
Data type                        int64              float64   
Null values count                    0                    2   

                  Malelifeexpectancy Femalelifeexpectancy District_x  \
Data type                    float64              float64     object   
Null values count                  2                    2          2   

                  Ward name Wardcode Crimerate Openspace Children  \
Data type            object   object   float64   float64  float64   
Null values count         2        0         0         0       66   

                  Greaterthan65 nonwhite NotBorninUK NotEnglishspeaking  \
Data type               float64  float64     float64            float64   
Null values count            66       66          66                 66   

                  District_y  
Data type             object  
Null values count         66

In [92]:
envi_health_demo[envi_health_demo['Greaterthan65'].isna()]

Population2011Census  GeneralFertilityRate  Malelifeexpectancy  \
3                   10800                  58.1                78.2   
8                   10286                  35.7                92.7   
11                  11991                  56.0                85.6   
26                  15113                  63.5                79.8   
37                  13389                  70.6                78.1   
..                    ...                   ...                 ...   
696                  9270                  30.9                89.9   
697                 14507                  52.6                83.0   
698                 15906                  48.9                80.5   
708                 12482                  64.2                77.2   
710                 10317                  68.8                78.3   

     Femalelifeexpectancy              District_x  \
3                    82.8                  Bexley   
8                    93.7           Tower Hamlets   
11                   88.7             Westminster   
26                   83.6                 Bromley   
37                   85.9  Hammersmith and Fulham   
..                    ...                     ...   
696                  93.4             Westminster   
697                  85.6                 Bromley   
698                  86.7                 Bromley   
708                  82.0                Havering   
710                  86.4    Richmond upon Thames   

                               Ward name Wardcode  Crimerate  Openspace  \
3                 Falconwood and Welling   00ADGN       44.2        0.7   
8            Spitalfields and Banglatown   00BGGL      213.5        1.4   
11           Bryanston and Dorset Square   00BKGC      255.3        2.1   
26             Bromley Common and Keston   00BBGN       85.5        3.2   
37               Wormholt and White City   00ANGR       79.2        4.2   
..                                   ...      ...        ...        ...   
696          Knightsbridge and Belgravia   00BKGH      184.9       69.1   
697         Chelsfield and Pratts Bottom   00AFGH       32.7       69.2   
698                 Hayes and Coney Hall   00AFGS       38.0       69.5   
708               Rainham and Wennington   00ARGQ       81.8       79.0   
710  Ham, Petersham & Richmond Riverside   00BDFZ       65.8       81.9   

     Children  Greaterthan65  nonwhite  NotBorninUK  NotEnglishspeaking  \
3         NaN            NaN       NaN          NaN                 NaN   
8         NaN            NaN       NaN          NaN                 NaN   
11        NaN            NaN       NaN          NaN                 NaN   
26        NaN            NaN       NaN          NaN                 NaN   
37        NaN            NaN       NaN          NaN                 NaN   
..        ...            ...       ...          ...                 ...   
696       NaN            NaN       NaN          NaN                 NaN   
697       NaN            NaN       NaN          NaN                 NaN   
698       NaN            NaN       NaN          NaN                 NaN   
708       NaN            NaN       NaN          NaN                 NaN   
710       NaN            NaN       NaN          NaN                 NaN   

    District_y  
3          NaN  
8          NaN  
11         NaN  
26         NaN  
37         NaN  
..         ...  
696        NaN  
697        NaN  
698        NaN  
708        NaN  
710        NaN  

[66 rows x 15 columns]

In [93]:
df_demographics[df_demographics['District'] == "Bexley"]

Children  Greaterthan65  nonwhite  NotBorninUK  NotEnglishspeaking  \
24   20.923674           15.6      31.0         23.3                 5.0   
40   15.815197           20.0      10.1         10.6                 2.2   
61   21.690718           14.6      11.7         10.0                 1.9   
72   16.299559           21.7       9.4          9.9                 2.1   
171  20.513964           17.6      11.0          9.9                 2.3   
196  27.162229            6.8      52.7         38.1                 7.6   
234  19.111111           15.7      15.4         12.4                 2.6   
247  23.637269           11.7      31.4         23.1                 5.6   
275  19.659005           17.5       7.3          7.1                 1.2   
294  18.501473           17.5      16.2         13.2                 3.1   
298  20.617098           15.1      14.5          9.8                 1.9   
311  18.967172           18.2       9.8          8.0                 1.3   
320  18.139358           21.0       8.4          8.1                 1.1   
329  26.802698           10.6      27.0         18.8                 3.3   
377  17.248246           20.9      13.1         10.4                 1.6   
433  22.354920           12.7      28.4         20.4                 5.1   
452  22.187275           14.5      17.7         12.3                 2.0   
460  18.916928           18.0      16.9         13.4                 3.2   
466  19.532539           17.9       9.5          7.8                 1.7   
467  18.103055           18.3      13.4         11.5                 2.3   
586  22.146385           13.7      17.3         11.6                 2.4   

    District             Ward name  
24    Bexley          Lesnes Abbey  
40    Bexley                Sidcup  
61    Bexley              Crayford  
72    Bexley             Longlands  
171   Bexley          Cray Meadows  
196   Bexley       Thamesmead East  
234   Bexley  Falconwood & Welling  
247   Bexley                 Erith  
275   Bexley   Blackfen & Lamorbey  
294   Bexley           Danson Park  
298   Bexley  Northumberland Heath  
311   Bexley            Barnehurst  
320   Bexley          Saint Mary's  
329   Bexley             North End  
377   Bexley              Brampton  
433   Bexley             Belvedere  
452   Bexley               Colyers  
460   Bexley          East Wickham  
466   Bexley     Blendon & Penhill  
467   Bexley          Christchurch  
586   Bexley       Saint Michael's